packages/base/src/3dview/mainview.tsx

```diff
@@ -132,6 +132,20 @@ export class MainView extends React.Component<IProps, IStates> {
       rotationSnapValue: 10,
       transformMode: 'translate'
     };
+
+    this._model.getSettings().then(settings => {
+      this._jcadSettings = settings.composite as any;
```

Comment:
I would not leak JupyterLab settings to the main view, all the setting values and signals should be encapsulated inside the model.

Reply:
Thanks @trungleduc, done in [60f4ac3](https://github.com/jupytercad/JupyterCAD/pull/725/commits/60f4ac39d87ebefea06c22f7ccfe67320889c63c)